developer/src/kmc-copy/src/copier-messages.ts

```diff
@@ -7,6 +7,7 @@
 import { CompilerErrorNamespace, CompilerErrorSeverity, CompilerMessageSpec as m, CompilerMessageDef as def, CompilerMessageSpecWithException } from "@keymanapp/developer-utils";
 
 const Namespace = CompilerErrorNamespace.Copier;
+const SevVerbose = CompilerErrorSeverity.Verbose | Namespace;
```

Comment:
Does SevDebug need to be defined?

Reply:
Only if it is used ... which it isn't currently